community/server/src/main/java/org/neo4j/server/rest/transactional/TransactionHandle.java

```diff
@@ -395,7 +396,7 @@ private void executePeriodicCommitStatement(
                 output.notifications( result.getNotifications() );
                 closeContextAndCollectErrors(errors);
             }
-            catch ( KernelException | CypherException e )
+            catch ( KernelException | CypherException | AuthorizationViolationException e )
             {
                 errors.add( new Neo4jError( e.status(), e ) );
             }
```

Comment:
I think we need to catch AuthorizationViolationException in AuthorizationFilter.doFilter to cover all REST endpoints, but it could also be done in a separate PR.